export function foo(): number {
  return 100
}

try {
  foo()
} catch (err) {}
